### DISCUSSION TITLE
Having problem while using the Spectral v6.

### DISCUSSION THREAD

**Discussion by adetroja** [stoplightio/spectral#2167] [Q&A]
● Test suite failed to run

    Cannot find module 'nimma/legacy' from 'node_modules/@stoplight/spectral-core/dist/runner/runner.js'

    Require stack:
      node_modules/@stoplight/spectral-core/dist/runner/runner.js
      node_modules/@stoplight/spectral-core/dist/runner/index.js
      node_modules/@stoplight/spectral-core/dist/spectral.js
      node_modules/@stoplight/spectral-core/dist/index.js
      node_modules/@stoplight/spectral-functions/dist/alphabetical.js
      node_modules/@stoplight/spectral-functions/dist/index.js
      tests/rules/adv-delete-operation-should-not-have-body.test.js

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (node_modules/@stoplight/spectral-core/src/runner/runner.ts:9:1)

**Comment by P0lip** [comment]
You can upgrade to newer Jest that [supports package exports](https://jestjs.io/blog/2022/04/25/jest-28#packagejson-exports) or set a `moduleNameMapper` in `jest.config.js` like so

```ts
 moduleNameMapper: {
    '^nimma/fallbacks$': '<rootDir>/node_modules/nimma/dist/cjs/fallbacks/index.js',
    '^nimma/legacy$': '<rootDir>/node_modules/nimma/dist/legacy/cjs/index.js',
  },
  ```
